src/v2/Artsy/Relay/middleware/cache/Cache.ts

```diff
@@ -26,7 +26,9 @@ export class Cache {
   constructor(cacheConfig: CacheConfig) {
     this.cacheConfig = cacheConfig
     this.enableServerSideCache =
-      isServer && !this.cacheConfig.disableServerSideCache
+      process.env.ENABLE_SERVER_SIDE_CACHE === "true" &&
```

Comment:
This is kind of an odd one! You added this ENV var to the `config` file (which itself _cant_ be required here, as this file is isomorphic on the client, and `config` has some server-side secrets. This guard: https://github.com/artsy/force/blob/33fb6dfc6cf414e51d34e94d3f68c7444a0c020d/src/desktop/config.coffee#L156 should actually alert that if you did require it.).

Then the value is added to Sharify (which is the currently safe way to isomorphically refer to a needed config value), but in here you're using `process.env`. Should this be referred to from sharify instead?

Reply:
Re server / vs client -- for isomorphic files this line is a bit blurry since it gets executed once on the server (with `process.env` values); and then on the client, where `process.env` has been forcefully injected by [webpack](https://github.com/artsy/force/blob/e5bbb95e7935a6a96c1d7bc552c3b2bcb4155c18/webpack/envs/baseConfig.js#L76). 

Updated to use our `getENV` helper, which will use sharify (if on the client) or `process.env` on the server.

Reply:
Oh wow! Ok i did not realize that was happening with webpack. I sometimes find the isomorphic stuff (when you really _think_ about it), a little freaky 😎